Makefile.libretro

```diff
@@ -383,6 +383,22 @@ else ifeq ($(platform), emscripten)
    TARGET := $(TARGET_NAME)_libretro_$(platform).bc
    DEFINES += -std=gnu99 -DHAVE_LOCALE
 
+# GCW0
+else ifeq ($(platform), gcw0)
+   TARGET := $(TARGET_NAME)_libretro.so
+   fpic := -fPIC
+   SHARED := -shared -Wl,-version-script=link.T
+   CC = /opt/gcw0-toolchain/usr/bin/mipsel-linux-gcc
+   AR = /opt/gcw0-toolchain/usr/bin/mipsel-linux-ar
+   PLATFORM_DEFINES += -D_GNU_SOURCE
+   CFLAGS += -fomit-frame-pointer -ffast-math -march=mips32 -mtune=mips32r2 -mhard-float
+   # The core is very slow on GCW0 platforms
+   # Use all possible optimisations...
+   CFLAGS += -finline-limit=42 -fno-unroll-loops -fno-ipa-cp
+   CFLAGS += -fno-common -fno-stack-protector -fno-guess-branch-probability
+   CFLAGS += -fno-caller-saves -fno-tree-loop-if-convert -fno-regmove
```

Comment:
Several of these seem to be disabling optimizations? Did you test one by one to make sure they all help?

Reply:
MIPS CPUs are very strange, and the optimisation strategies are often counter intuitive.

These (de-)optimisations do in fact help - but the fact that you raised concerns about them makes me regard this as a real issue. I imagine that most devs would look at that mess of unsavoury flags and think exactly the same thing - leading to confusion and a maintenance burden. The performance difference isn't really significant (not when so few GBA games run full speed anyway) - so on reflection, I don't think it's worth the hassle.

I've therefore updated the PR to remove all these non-standard flags, and instead have just included the following 'uncontroversial' ones, which are of some small benefit in isolation:

```
CFLAGS += -fno-common -ftree-vectorize -funswitch-loops
```

(I realise that `-ftree-vectorize` and  `-funswitch-loops` are enabled by default at `-O3`, but I want to make sure they are there in case the global optimisation level ever changes - or updates to the GCW0 toolchain change something...)

The core will never be practical for GBA games on OpenDingux (not without a dynarec!), but it turns out that mGBA is still of great use on the platform - it seems to provide the best method of playing SGB-enhanced GB titles at full speed (with 3x upscaling) on an RG350M, which is not to be sniffed at!